apis/v1beta1/httproute_types.go

```diff
@@ -429,6 +429,13 @@ type HTTPQueryParamMatch struct {
 	// entry with an equivalent name MUST be considered for a match. Subsequent
 	// entries with an equivalent query param name MUST be ignored.
 	//
+	// If a query param is repeated in an HTTP request, it is
+	// implementation-specific behavior as to how this is represented. However,
+	// implementations should match against the first value of the param if
+	// the data plane supports it.
```

Comment:
This seems a little unclear to me, how about this?
```suggestion
	// If a query param is repeated in an HTTP request, the behavior
	// is purposely left undefined, since different data planes have different
	// capabilities. However, it's *recommended* that implementations should
	// match against the first value of the param if the data plane supports it,
	// as this behavior is expected in other contexts.
```
(Sorry, I have had no bandwidth starting the new job).

Reply:
@youngnick 

Good suggestion 👍 

> as this behavior is expected in other contexts.

This looks a little unclear to me.  Do you imply any similar Gateway API guidelines?

Reply:
Ah, sorry, I missed this. By "in other contexts", I meant "in other, non-Gateway API load balancing contexts".

Reply:
@youngnick thanks! I applied your suggestion with added clarification about load balancing contexts.